pkgs/development/tools/parsing/spicy/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  bison,
+  flex,
+  zlib,
+  python3
+}:
+let
+  version = "1.5.2";
+in stdenv.mkDerivation {
+  pname = "spicy";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "zeek";
+    repo = "spicy";
+    rev = "v${version}";
+    hash = "sha256-HcSDHmPRwfdG0GAsrT4CWvXi6s6xuwHLmHaKDUKlgbM=";
+    fetchSubmodules = true;
```

Comment:
Please use our libraries instead https://github.com/zeek/spicy/blob/main/.gitmodules

Reply:
7 or 13 are currently not in nixpkgs. Given that it mostly copies specific 3rdparty headers into its own output I would prefer to take the easy route and avoid the maintenance burden.

Reply:
On first look does not look like anything to security related is under the packages.